netbox/ipam/tables.py

```diff
@@ -169,8 +169,12 @@
 """
 
 TENANT_LINK = """
```

Comment:
Let's leave this as-is for now, and take another look at how we can efficiently include tenant groups in object lists under a separate issue.

Reply:
Removed the Tenant Groups